LEAF_Request_Portal/admin/css/mod_groups.css

```diff
@@ -84,4 +84,52 @@ div.groupBlockWhite h3 {
     font-weight: normal;
     font-family: 'PublicSans-Medium';
 
+}
+
+/* Loading Spinner CSS */
```

Comment:
Shouldn't need CSS comments as the class itself should describe what it does.  Please convert to describe classes better.  Also, CSS classes are not usually camelcase.  They should be words connected by dashes (and usually no longer than three words if you can).

Reply:
Thanks for the clarification, will update.